plugin

```diff
@@ -153,6 +153,7 @@
   "jtbgroup/kodi-playlist-card",
   "jtbgroup/kodi-search-card",
   "junalmeida/homeassistant-minimalistic-area-card",
+  "junalmeida/homeassistant-webrtc-card",
```

Comment:
![image](https://user-images.githubusercontent.com/15093472/218328175-d3c81ebb-a483-455e-9859-f8bdba8df1cf.png)
Does not show the card at all.
<https://hacs.xyz/docs/publish/include#check-images>